xml/System/Enum.xml

```diff
@@ -1981,9 +1995,9 @@ thisInstance And flag = flag
         <Parameter Name="provider" Type="System.IFormatProvider" Index="0" FrameworkAlternate="dotnet-uwp-10.0;netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.5;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="provider">An <see cref="T:System.IFormatProvider" /> implementation that supplies culture-specific information about the format of the returned value. This argument is not used.</param>
+        <summary>This method overload is obsolete; please use <see cref="M:System.Enum.ToString()" />.</summary>
```

Comment:
Why the API is not officially marked as obsolete then? That would make the obsolete text appear automatically.

```suggestion
        <summary>This method overload is obsolete; use <see cref="M:System.Enum.ToString" /> instead.</summary>
```